client/gutenboarding/controller.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import config from 'config';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { Gutenboard } from './gutenboard';
+import { hideMasterbar as hideMasterbarAction } from 'state/ui/actions';
+
+export function redirectIfNotEnabled( context: PageJS.Context, next ) {
+	if ( ! config.isEnabled( 'gutenboarding' ) ) {
+		page.redirect( '/' );
+		return;
+	}
+	next();
+}
+
+export function hideMasterbar( context: PageJS.Context, next ) {
+	context.store.dispatch( hideMasterbarAction() );
+	next();
+}
+
+export function main( context: PageJS.Context, next ) {
```

Comment:
These can also be defined as follows and we don't need to provide return or param types:

```suggestion
export const main: PageJS.Callback = ( context, next ) => {
```

Reply:
I'll make this change